java-native/src/test/java/com/baeldung/unsatisfiedlink/JniUnsatisfiedLinkManualTest.java

```diff
@@ -0,0 +1,94 @@
+package com.baeldung.unsatisfiedlink;
+
+import static com.baeldung.unsatisfiedlink.JniUnsatisfiedLink.LIB_NAME;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+
+import org.junit.jupiter.api.Test;
+
+public class JniUnsatisfiedLinkManualTest {
+
+    @Test
+    public void whenCorrectLibName_thenLibLoaded() {
+        assertDoesNotThrow(() -> {
+            System.loadLibrary(LIB_NAME);
+            new JniUnsatisfiedLink().test();
+        });
+    }
+
+    @Test
+    public void whenIncorrectLibName_thenLibNotFound() {
+        String libName = "lib" + LIB_NAME + ".so";
+
+        Error error = assertThrows(UnsatisfiedLinkError.class, () -> {
+            System.loadLibrary(libName);
+        });
```

Comment:
For instance, of a replacement of lambda statement for lambda expression:
`Error error = assertThrows(UnsatisfiedLinkError.class, () -> System.loadLibrary(libName));`